### PR TITLE
Raeshawn bart code review

### PR DESCRIPTION
**Pull request by bartrn**
analysis of program: to be able to keep track of appointment and prescriptions. most things currently aren't implemented.

Does the program compile: yes the program compiles , with some warnings.

was it available on GitHub on a timely manner: yes 

is the program well documented and commented well:yes 

rationale behind changes: gave suggestion on how to fix the main activity warning. the naming conventions are very well done left minor notes 

What i learned: how to do a notification. other then this most of code is not implemented yet to learn something further

### CODE REVIEW

File: app/src/main/java/com/medprompt/HomeActivity.kt [hunk 1]
```diff
@@ -30,7 +30,8 @@ class HomeActivity : ComponentActivity() {
         }
     }
 }
-
+// work putting space between the ) and the } to resolve that warning 
+//nice implementations of vars would work on naming such as capitalizing first letter
```
Comment (discospiff) [inline app/src/main/java/com/medprompt/HomeActivity.kt:34]: Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.

File: app/src/main/java/com/medprompt/HomeActivity.kt [hunk 2]
```diff
@@ -40,8 +41,10 @@ fun ToDoItemFacts(name: String) {
     var stock by remember { mutableStateOf("")}
     val context = LocalContext.current
 
+    //fixed value to another line to align
```
Comment (discospiff) [inline app/src/main/java/com/medprompt/HomeActivity.kt:44]: Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.

File: app/src/main/java/com/medprompt/dao/INotificationDAO.kt [hunk 3]
```diff
@@ -1,5 +1,5 @@
 package com.medprompt.dao
-
+// learned that this is how you make notifications
```
Comment (discospiff) [inline app/src/main/java/com/medprompt/dao/INotificationDAO.kt:2]: Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.

**Comment by discospiff** [comment]
Since this pull request contains mostly discussion in code, I don't recommend merging it.

File: app/src/main/java/com/medprompt/HomeActivity.kt (hunk 1 above)
Comment (alexspurlock25) [reply app/src/main/java/com/medprompt/HomeActivity.kt:34]: Kotlin's conventions says that we should use camel case for vars

**Pull request closed without merging**